### ISSUE TITLE
Fail to build docker image with latest GoLang version

### ISSUE THREAD

**Issue by JoshuaMcManus** [FoundationDB/fdb-kubernetes-operator#252]
When running the `make samples docker-build` command I'm getting the following error:

```
Step 6/27 : RUN 	curl $FDB_WEBSITE/downloads/$FDB_VERSION/ubuntu/installers/foundationdb-clients_$FDB_VERSION-1_amd64.deb -o fdb.deb && 	dpkg -i fdb.deb && rm fdb.deb && 	for version in ${FDB_VERSION} ${FDB_ADDITIONAL_VERSIONS}; do 		minor=${version%.*} && 		mkdir -p /usr/bin/fdb/$minor/tmp && 		curl $FDB_WEBSITE/downloads/$version/linux/fdb_$version.tar.gz -o /usr/bin/fdb/$minor/binaries.tar.gz && 		tar --strip-components=1 -C /usr/bin/fdb/$minor/tmp -xzf /usr/bin/fdb/$minor/binaries.tar.gz && 		rm /usr/bin/fdb/$minor/binaries.tar.gz && 		for binary in fdbcli fdbbackup fdbrestore; do 			mv /usr/bin/fdb/$minor/tmp/$binary /usr/bin/fdb/$minor/$binary; 		done && 		rm -r /usr/bin/fdb/$minor/tmp && 		chmod u+x /usr/bin/fdb/$minor/fdb*; 	done && 	mkdir -p /usr/lib/fdb && 	for version in ${FDB_ADDITIONAL_VERSIONS}; do 		curl $FDB_WEBSITE/downloads/$VERSION/linux/libfdb_c_$VERSION.so -o /usr/lib/fdb/libfdb_c_$VERSION.so; 	done
 ---> Running in 3c79e8478602
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (60) SSL certificate problem: unable to get local issuer certificate
More details here: https://curl.haxx.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.
```

It seems to go away if I set the Go version to 1.13.11 explicitly. We'll want to investigate this to see if there's some sort of process change when pulling a file from the foundationdb servers.

**Comment by brownleej** [comment]
The golang image includes the ca-certificates package, and that package was updated in early June to remove the root CA that our website is using. Changelog for that package: https://metadata.ftp-master.debian.org/changelogs//main/c/ca-certificates/ca-certificates_20200601_changelog

I think this is a consequence of this change from Mozilla: https://blog.mozilla.org/security/2018/03/12/distrust-symantec-tls-certificates/. Though that blog post only discusses changes in 2018, I see a more recent change in the bug tracker that seems to match the change in the ca-certificates package: https://bugzilla.mozilla.org/show_bug.cgi?id=1618402

We could work around this by trying to revert the CA package, or by manually installing the GeoTrust Global CA in our build image. It may be better to try to get a new certificate that is coming from a CA that is still trusted by the latest package.

**Comment by rbtcollins** [comment]
Building with 1.13.2 proceeds further, I'll have a PR up shortly that fixes this and another issue ( version vs VERSION) as well.

**Comment by rbtcollins** [comment]
The root CA your website uses shouldn't have been removed: the documentation in the ticket says it was updated to remove the email trust bit.

```
    Disable the Email trust bit for the following root certs. (i.e. set CKA_TRUST_EMAIL_PROTECTION to CK_TRUST CKT_NSS_MUST_VERIFY_TRUST)

    Subject: CN=GeoTrust Global CA; O=GeoTrust Inc.; C=US
    Certificate Serial Number: 023456
    SHA-1 Fingerprint: DE28F4A4FFE5B92FA3C503D1A349A7F9962A8212
    SHA-256 Fingerprint: FF856A2D251DCD88D36656F450126798CFABAADE40799C722DE4D2B5DB36A73A
```

on the other hand, the particular geotrust root cert is not in the list of whitelisted certs: 
https://wiki.mozilla.org/CA/Additional_Trust_Changes#Symantec:
```
    c0554bde87a075ec13a61f275983ae023957294b454caf0a9724e3b21b7935bc
    56e98deac006a729afa2ed79f9e419df69f451242596d2aaf284c74a855e352e
    7289c06dedd16b71a7dcca66578572e2e109b11d70ad04c2601b6743bc66d07b
    fae46000d8f7042558541e98acf351279589f83b6d3001c18442e4403d111849
    b5cf82d47ef9823f9aa78f123186c52e8879ea84b0f822c91d83e04279b78fd5
    e24f8e8c2185da2f5e88d4579e817c47bf6eafbc8505f0f960fd5a0df4473ad3
    3174d9092f9531c06026ba489891016b436d5ec02623f9aafe2009ecc3e4d557
```

Your root certs sha256 hash is FF:85:6A:2D:25:1D:CD:88:D3:66:56:F4:50:12:67:98:CF:AB:AA:DE:40:79:9C:72:2D:E4:D2:B5:DB:36:A7:3A

so perhaps the second phase is happening ?

I propose to build with 1.13.11 in the interim, to unblock things, the generated go code should be fine, the only go 1.13 CVE is for much lower versions. https://www.cvedetails.com/vulnerability-list/vendor_id-14185/product_id-29205/version_id-352992/Golang-GO-1.13.html

**Comment by brownleej** [comment]
I think we should leave this open until we can build with the latest golang image.
